spec/steps/claim_details/form_spec.rb

```diff
@@ -46,6 +56,8 @@
 
     context 'when preparation is yes' do
       let(:preparation_time) { 'yes' }
+      let(:work_before_date) { Date.new(2023, 1, 1) }
+      let(:work_after_date) { Date.new(2023, 1, 1) }
```

Comment:
these additional lets are not required here as they are already added at the top level